openslide/__init__.py

```diff
@@ -438,14 +463,16 @@ def read_region(self, location, level, size):
         ]:  # "< 0" not a typo
             # Crop size is greater than zero in both dimensions.
             # PIL thinks the bottom right is the first *excluded* pixel
-            crop = self._image.crop(image_topleft + [d + 1 for d in image_bottomright])
-            tile_offset = tuple(il - l for il, l in zip(image_topleft, location))
+            crop = self._image.crop(
```

Comment:
crop expects a 4-tuple rather than an n-tuple

Reply:
Likewise, we can just assert these lengths, so I've reverted this.